benchmark/json_schema/bench_sglang.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import json
+import time
+
+import jsonschema
+from datasets import load_dataset
+
+import sglang as sgl
+from sglang.lang.chat_template import get_chat_template_by_model_path
+from sglang.test.test_utils import (
+    add_common_sglang_args_and_parse,
+    select_sglang_backend,
+)
+from sglang.utils import dump_state_text
+
+
+@sgl.function
+def schema_gen(s, message: str, json_schema: str):
+    s += message
+    s += sgl.gen("json_output", temperature=0, max_tokens=256, json_schema=json_schema)
+
+
+def convert_dataset(path: str):
+    tmpl = get_chat_template_by_model_path("Llama-3.1-8B-Instruct")
```

Comment:
You do not need this. You can use https://github.com/sgl-project/sglang/blob/a10d5309436f83c34c1dc948f5601dd8895e7df2/python/sglang/test/test_programs.py#L45-L46